### PR TITLE
v0.13.1rc3 gitian sigs from midnightmagic

### PR DESCRIPTION
**Pull request by midnightmagic**
So there.

### CODE REVIEW

File: 0.13.1rc3-win/midnightmagic/bitcoin-win-0.13-build.assert [hunk 1]
```diff
@@ -0,0 +1,1175 @@
...
+
+    c039ffd98c99e6d2645e402d017c7e4eff79d9fd3e6b644f9379f670352d68e1  x86_64-w64-mingw32/boost/boost-1_59_0-68550199124.tar.gz.hash
+
+    648b5a3c93b925dfc1e4baf076569f1318634e4dda1016e010f11060a0efb9cf  x86_64-w64-mingw32/boost/boost-1_59_0-b0bbbf17cd7.tar.gz
+
+    99a54ff58a2ba1e046d94d11a4593ca08874eb2004f18bb53f1a6ce4e0b2f368  x86_64-w64-mingw32/boost/boost-1_59_0-b0bbbf17cd7.tar.gz.hash
+
+    bb0995603d28363393170d7c4384bd40fe2d31a43a9108275b1ba573726f5cb9  x86_64-w64-mingw32/libevent/libevent-2.0.22-7ece0cb62c3.tar.gz
+
+    aa4477d45710e6192287c64e181ca69ada600b2dee1ef95016d3c238a54bc300  x86_64-w64-mingw32/libevent/libev
```
Comment (Michagogo) [inline 0.13.1rc3-win/midnightmagic/bitcoin-win-0.13-build.assert]: Should be -win-unsigned.

**Review by Michagogo — CHANGES_REQUESTED** [review]
Should be -win-unsigned.

**Comment by midnightmagic** [comment]
Woopsie -- yes of course. Thanks for the catch.

**Comment by Michagogo** [comment]
This will fail to validate. You fixed the folder name, but that's not enough -- you also need to change the "release" tag at the bottom and resign.

**Pull request merged**